llamafile/sgemm.cpp

```diff
@@ -128,3 +134,8 @@ bool llamafile_mixmul(const ggml_compute_params *params, const ggml_tensor *weig
                       const ggml_tensor *thought, const ggml_tensor *plan, ggml_tensor *result) {
     return funcs.mixmul(params, weights, thought, plan, result);
 }
+
+bool llamafile_mixmul_iqk(long Nx, long Ny, long ne00, int ne11, int typeA, const void * A, const void * B,
+        float * C, long nb1, long nb2, const void * vrow_mapping, int ith, int nth) {
+    return funcs.iqk_mixmul ? funcs.iqk_mixmul(Nx, Ny, ne00, ne11, typeA, A, B, C, nb1, nb2, vrow_mapping, ith, nth) : false;
```

Comment:
For consistency with the other delegates, could I request a no-op implementation of this function that returns false? You could put it in `llamafile/tinyblas_cpu_unsupported.cpp`.